posts/verify-sync-release-volunteers/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Call for volunteers: help to test us the release syncing using staging instance"
+date: 2023-08-30T14:46:18+02:00
+authors: lbarczio
+tags:
+  - downstream
+  - pull-from-upstream
+  - propose-downstream
+  - staging
+---
+
+In the upcoming months, we plan to migrate our service to a new cluster. However, this may affect `propose_downstream`
+and `pull_from_upstream` jobs due to the new firewall rules. The problematic aspects could be:
+
+- commands you run in your `actions` during syncing the release involving interactions with external servers
+- downloading your sources from various hosting services (crates.io, npm, gems, etc.)
+
+To smoothen this transition, we kindly encourage you to enable one of these jobs on our already migrated staging instance.
+This recommendation is particularly important if you belong to one of the groups affected by the two previous points.
+This proactive step will help us identify and address any issues promptly. To avoid too much noise in your dist-git PRs,
+you may enable the job for only one target, resulting in only one additional PR created.
```

Comment:
I would explicitly mention, that you can run both instances together -- and add the new one but still preserve the production one. (It's somehow mentioned later on, but people not familiar with how our staging instance works might be confused.)

Reply:
added